code/UserForm.php

```diff
@@ -344,6 +264,89 @@ public function getCMSFields()
         return $fields;
     }
 
+    public function getSubmissionsGridField()
```

Comment:
Not sure whether this should be 
```suggestion
    protected function getSubmissionsGridField()
```

Could even be `private` if there's an update extension point added before the gridfield is returned.

Reply:
There's no need for a hook specifically for this, `getCMSFields` has one. This is expanding PR scope. If someone finds it to be a bother then it can be added by that person easily in another PR.

I left it public so it could be called directly by e.g. an extension that moves the submissions to it's own separate page-load tab (like 'Settings' - c.f. `SiteTree::getSettingsFields`).